package.json

```diff
@@ -91,6 +91,7 @@
     "yn": "^2.0.0"
   },
   "resolutions": {
+    "socket.io/**/debug": "^3.2.6",
```

Comment:
Since it doesn't impact the issue on frontend with ie11, please remove this one, it's not necessary